source/extended-json.rst

```diff
@@ -0,0 +1,618 @@
+=============
+Extended JSON
```

Comment:
Nowhere in this spec is the term `extjson` used, while it is extensively used in the "bson corpus" and its tests, which includes the tests for this spec.